cfme/utils/template/ec2.py

```diff
@@ -71,6 +71,10 @@ def run(self):
         if not self.download_image():
             return False
 
+        if 'miq' in self.template_name:
```

Comment:
if `unzip_image()` detects the file type and only unpacks when appropriate, then there is no need to check for template name substrings to explicitly call this. Combined with it being called by download_image and there's no need for this additional call here.

Reply:
Done